src/config/config.cc

```diff
@@ -723,7 +726,13 @@ void Config::Get(std::string key, std::vector<std::string> *values) {
   for (const auto &iter : fields_) {
     if (key == "*" || Util::ToLower(key) == iter.first) {
       values->emplace_back(iter.first);
-      values->emplace_back(iter.second->ToString());
+      if (iter.second->GetConfigType() == configType::MultiConfig) {
+        for (const auto &p : Util::Split(iter.second->ToString(), "\n")) {
+          values->push_back(p);
+        }
+      } else {
+        values->emplace_back(iter.second->ToString());
+      }
```

Comment:
```suggestion
      if (iter.second->GetConfigType() == configType::MultiConfig) {
        for (const auto &p : Util::Split(iter.second->ToString(), "\n")) {
          values->emplace_back(iter.first);
          values->push_back(p);
        }
      } else {
        values->emplace_back(iter.first);
        values->emplace_back(iter.second->ToString());
      }
```
I think we need to keep one-key-one-value pairs, in order to make user to parse the result of `config get *` easily.